### PR TITLE
323/elewa brands italanta

### PR DESCRIPTION
**Pull request by Anna-Ashton**
# Description

This pull request is a feature creation of the elewa brands italanta, we worked with @[chegeian](https://github.com/chegeian) and @[littlephillips](https://github.com/littlephillips). 

To achieve this, we needed to:
   create a
 1. Brand Logo
 2. Brand Name
 3. Brands Description
 4. Button
 by reusing [#321](https://github.com/italanta/elewa-group/issues/321)

Fixes [#323](https://github.com/italanta/elewa-group/issues/323)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# Screenshot (optional)
Screenshot from 2023-02-27 20-49-30

#Screenshot media responsive
italanta-brands

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Anna-Ashton** [comment]
@IanOdhiambo9 kindly review the above

File: libs/pages/elewa/brands/src/lib/pages-elewa-brands.module.ts [hunk 1]
```diff
@@ -9,9 +9,14 @@ import { ElewaBrandsOpportunitiesComponent } from './components/elewa-brands-opp
 import { ElewaBrandsPageComponent } from './pages/elewa-brands-page/elewa-brands-page/elewa-brands-page.component';
 
 import { BrandsRoutingModule } from './brands.routing';
+import { ElewaBrandsItalantaComponent } from './components/elewa-brands-italanta/elewa-brands-italanta.component';
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/brands/src/lib/pages-elewa-brands.module.ts:12]: order of imports

router should be imported last and pages are imported after components

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
Resolve conflicts

File: libs/pages/elewa/brands/src/lib/pages-elewa-brands.module.ts (hunk 1 above)
Comment (Anna-Ashton) [reply libs/pages/elewa/brands/src/lib/pages-elewa-brands.module.ts]: @IanOdhiambo9 Kindly review, did correction on order of imports

**Comment by Anna-Ashton** [comment]
@IanOdhiambo9 kindly review the corrections

**Comment by IanOdhiambo9** [comment]
This issue has been closed

**Pull request closed without merging**